modules/mod_breadcrumbs/tmpl/default.php

```diff
@@ -74,12 +74,28 @@
             'itemListElement' => []
     ];
 
+    // Use an independent counter for positions. E.g. if Heading items in pathway.
+    $itemsCounter = 0;
+
+    // If showHome is disabled use the fallback $homeCrumb for startpage at first position.
+    if (isset($homeCrumb))
+    {
```

Comment:
```suggestion
    if (isset($homeCrumb)) {
```
Please fix code style (we meanwhile use PSR-123 standard) to fix the PHPCS check in drone CI.